be/src/storage/tablet_meta_manager.cpp

```diff
@@ -804,7 +804,8 @@ Status TabletMetaManager::rowset_iterate(DataDir* store, TTabletId tablet_id, co
 Status TabletMetaManager::apply_rowset_commit(DataDir* store, TTabletId tablet_id, int64_t logid,
                                               const EditVersion& version,
                                               vector<std::pair<uint32_t, DelVectorPtr>>& delvecs,
-                                              const PersistentIndexMetaPB& index_meta, bool enable_persistent_index) {
+                                              const PersistentIndexMetaPB& index_meta, bool enable_persistent_index,
+                                              Rowset* rowset) {
```

Comment:
passing a rowset ptr without meaning seems confusing, is the intention to persist rowset meta pb or `clear rowset txn meta then persist if rowset not empty`? the intention is just to complex. 
I think passing a rowset meta pb(or null) is more clear.

Reply:
> passing a rowset ptr without meaning seems confusing, is the intention to persist rowset meta pb or `clear rowset txn meta then persist if rowset not empty`? the intention is just to complex. I think passing a rowset meta pb(or null) is more clear.

done, changed in https://github.com/StarRocks/starrocks/pull/13285/commits/0c44e1b2804661258d1d9495f390201d4caa5f94

Reply:
run starrocks_be_unittest